rpc-tests/tests/0001-register-xcasset.js

```diff
@@ -0,0 +1,38 @@
+async function run(nodeName, networkInfo, args) {
+    networkInfo.nodesByName["astar"].hrmpTo = 2007
+    networkInfo.nodesByName["shiden"].hrmpTo = 2006
+
+    const { sendTransaction } = await import("./tx-utils.mjs");
+    const { wsUri, userDefinedTypes, hrmpTo } = networkInfo.nodesByName[nodeName];
+    const api = await zombie.connect(wsUri, userDefinedTypes);
+
+    await zombie.util.cryptoWaitReady();
+
+    // account to submit tx
+    const keyring = new zombie.Keyring({ type: "sr25519" });
+    const sender = keyring.addFromUri("//" + args[0]);
+
+    const tx0 = await api.tx.assets.forceCreate(hrmpTo, {id: sender.address}, true, 1);
+
+    const assetLocation = `{"v1":{"parents":1,"interior":{"x1":{"parachain":${hrmpTo}}}}}`;
```

Comment:
Wouldn't it be better to use the _object_ approach instead?

These large strings tend to become unreadable (and caused you issues before).

Reply:
Comparing with Zombienet [tutorial](https://docs.astar.network/docs/build/environment/zombienet-testing#basic-cross-chain-assets-setup-instructions) step1 I miss AssetId here

Reply:
One reason I did it like a string here is that I can use it as an input and compare it to the output of the `assetIdToLocation` as a string.  When I tried to do the same comparison with the objects, they were quite different. 

Aslo Mario, I've added assetId variable for clarity

Reply:
In general, using strings in the code to pass around information is bad practice (I think this applies to every language).

It's easier to construct objects, then large strings. Also if we create some helper methods for e.g. creating XCM messages, it easier for them to construct _objects_, instead of strings.

For comparison, I think you can just stringify the object, if you want to compare them visually.